app/models/rules/sealed_bid.rb

```diff
@@ -0,0 +1,53 @@
+module Rules
+  class SealedBid < Rules::Basic
+    def winning_bid
+      return Presenter::Bid::Null.new if auction.available?
```

Comment:
I imagine NullBid could be a plain Ruby object that is used to prevent nil
Vals in calculations rather than views.

We don't have one of these NullBids yet.

Reason I am suggesting change here is that I think it makes sense for view
objects or presenters to return presenters, but not things in "app/models",
which I would assume to contain "business logic" rather than "view logic".

Does my thinking make sense to you? Maybe we can have a quick chat about it
if not bc I am curious in your thoughts too!

On Monday, April 25, 2016, Jacob Harris <notifications@github.com> wrote:

> In app/models/rules/sealed_bid.rb
> <https://github.com/18F/micropurchase/pull/419#discussion_r61017996>:
>
> > @@ -0,0 +1,53 @@
> > +module Rules
> > +  class SealedBid < Rules::Basic
> > +    def winning_bid
> > +      return Presenter::Bid::Null.new if auction.available?
>
> But I will admit I'm not entirely sure of the difference between
> Presenter::Bid::Null and NullBid
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/18F/micropurchase/pull/419/files/b8d7eba86aa3012205d4077f35ebc222c388c83a#r61017996>
>